deploy/deploy_ancillaryRH7.dev_local

```diff
@@ -0,0 +1,144 @@
+      shell: pip install gdown  
+#     yum:
+#       name: python27-python-pip
+#       state: latest
+    - name: Install python 3.6 and pip
+      shell: yum install -y python36 python36-pip
+
+#   - name: enable scl
+#     become: yes
+#     become_method: sudo
+#     shell: scl enable python27 bash
+#   - name: install gdown
+#     shell: /opt/rh/python27/root/usr/bin/pip install gdown
+
+#     Already secure copy to destination host
+    - name: use gdown to download and build local repo
+      shell: gdown https://drive.google.com/uc?id=1qaZH9_SrGyjHjWT3P2WtvZAhEkP9YG1Z -O /tmp/ancilRH7_repo.tar.gz
+#     when: stage == "DEV_LOCAL"
+#       stage values = { "DEV_LOCAL","DEV", "PROD" }
+
+#     shell: gdown https://drive.google.com/uc?id=139Z0MfyT0WVWIzP_uwM2KxPCrfYs_vUV
+#   - name: Install pip
+#     easy_install:
+#       name: pip
+#       state: latest
+#    - name: Build custom local repo
+#      shell: wget https://drive.google.com/open?id=1qaZH9_SrGyjHjWT3P2WtvZAhEkP9YG1Z /tmp
+    - name: make localrepo directory
+      shell: mkdir -p /opt/ancillary_local
+    - name: make OpenOffice directory
+      shell: mkdir -p /opt/ancillary_local/OO416
+    - name: make Sublime3 directory
+      shell: mkdir -p /opt/ancillary_local/Sublime3
+    - name: make Sublime3 install directory
+      shell: mkdir -p /usr/local/sublime
+    - name: make stattransfer14 install directory
+      shell: mkdir -p /usr/local/StatTransfer14
+
+
+    - name: Extract repo to destination point
+      shell:  tar xvf /tmp/ancilRH7_repo.tar.gz -C /opt/ancillary_local
+    - name: Extract OpenOffice 4.1.6 
+      shell:  tar xvf /opt/ancillary_local/Apache_OpenOffice_4.1.6_Linux_x86-64_install-rpm_en-US.tar.gz -C /opt/ancillary_local/OO416 
+    - name: install bz2
+      yum:
+        name: bzip2
+
+    - name: Extract Sublime3 
+      shell:  tar xvf /opt/ancillary_local/sublime_text_3_build_3207_x64.tar.bz2 -C /opt/ancillary_local/Sublime3
+
+    - name: Extract Sublime3 to install dir
+      shell:  tar xvf /opt/ancillary_local/sublime_text_3_build_3207_x64.tar.bz2 -C /usr/local/sublime 
+
+    - name: install createrepo
+      yum:
+        name: createrepo
+        state: latest
+    - name: create local repo 
+      shell: createrepo /opt/ancillary_local/OO416
+#     shell: createrepo /opt/devrepo-master/Packages
+    - name: create local repo file with updated link
+      file:
+        path: /etc/yum.repos.d/local.repo
+        state: touch
+        mode: "u=rw,g=r,o=r" 
+    - name: edit local.repo file
+      shell: |
+        echo '[localrepo]
+        name=localrepo
+        baseurl=file:///opt/ancillary_local/OO416
+        enabled=1
+        gpgcheck=0' >> /etc/yum.repos.d/local.repo
+  roles:
+    - umask
+#   - devtools
+    - devtools7
+#   - open_office
+    - open_officeRH7
+#   - evince
+#   See line #39 (evince)
+#   - stattransfer
+    - stattransfer14
+#   - sublime
+    - sublime3
+#   - ultra-edit
+    - ultra-editRH7
+    - texliveRH7
```

Comment:
@lel99999 Why are we calling these roles for the local install, but not the prod deployment? Looks like in prod we're deploying each role as a task in the playbook. I think we prefer to break them into roles. 

Also, why are there two separate playbooks for local and prod - typically we've used the when: custom_repo tag to be a determination between prod and local development.

Reply:
just a marker to work out the diffs and conditionals before a single playbook covering multiple scenarios are integrated